packages/perseus-editor/package.json

```diff
@@ -59,6 +59,7 @@
         "@khanacademy/wonder-blocks-form": "catalog:",
         "@khanacademy/wonder-blocks-icon": "catalog:",
         "@khanacademy/wonder-blocks-icon-button": "catalog:",
+        "@khanacademy/wonder-blocks-labeled-field": "catalog:",
```

Comment:
I don't see any `perseus-editor` changes in this PR, so is adding this dependency here needed?